### DISCUSSION TITLE
Entering of numbers on TX16S

### DISCUSSION THREAD

**Discussion by wumfi** [EdgeTX/edgetx#5029] [Q&A] [answered]
I'm sure this has already been asked, but I'm struggling what to search for to find the answer.

When entering numbers in fields (such as the high/low value of a widget), the method seems inconsistent any often annoying.

For example: When setting up a gauge, you define a min and max value. Clicking in those fields either gives you direct access (A) to change the value with the scroll wheel, or pops up the << - + >> menu (B). Changing a large value, such as 1024 -> 10 takes forever with the B method.

I have 2 models with the same gauge widget, and one uses method A, the other method B.

Is there a way to toggle the method, or is there a bug that picks one randomly?

TIA!

**Comment by pfeerick** [comment, marked as answer]
Is it possible you are using touch to select the input field one time, and
the roller ENTER another? As selecting using touch will bring up the popup
keyboard, whereas ENTER will do direct entry. And this should be for *all*
number edit fields.

**Reply by wumfi** [reply]
That is exactly it! Thank you!

I’ve been mucking about all day trying to figure this out!
